chef/cookbooks/updater/recipes/default.rb

```diff
@@ -40,6 +40,31 @@
       zypper_command += " --auto-agree-with-licenses"
     end
 
+    ruby_block "check for HA updates" do
+      block do
+        case node[:updater][:zypper][:method]
+        when "patch"
+          command = "zypper list-patches"
+        when "update"
+          command = "zypper list-updates"
+        when "dist-upgrades"
+          # TODO: check the correct one for here
+          command = "zypper list-updates"
+        end
+        command += '|egrep "corosync|pacemaker" -q'
```

Comment:
According to the `grep` man page, `-q` should come before the pattern, not after.  Even if it's not necessarily, it's more legible because it's what people expect.